senseobjdef/constraint.go

```diff
@@ -192,3 +201,29 @@ func (operator constraintOperator) evalString(val string, constraint string) (bo
 		return false, errors.Errorf("operator<%s> not supported for string constraint evaluation", str)
 	}
 }
+
+func (operator constraintOperator) evalArray(val interface{}, constraint string) (bool, error) {
+	v := reflect.ValueOf(val)
+	n := v.Len()
+
+	switch operator {
+	case containsOperator:
+		for i := 0; i < n; i++ {
+			e := v.Index(i).Interface()
+			str, ok := e.(string)
```

Comment:
should maybe support ints and float arrays too? i.e. a switch on type then cast

Reply:
Sure, why not